src/humidity.toit

```diff
@@ -6,24 +6,23 @@ import system.services
 
 import .clients.humidity as clients
 
-SELECTOR ::= services.ServiceSelector
+SELECTOR-v1-0 ::= services.ServiceSelector
```

Comment:
The selector shouldn't change across versions?

Reply:
The selector changes across major versions, but the UUID doesn't.